app/css/obsadmin.css

```diff
@@ -518,3 +518,8 @@ li>a:hover {
     color: #FF0000;
     margin-left: 180px;
 }
+.obs-err-msg {
+    color: #FF0000;
+    margin-left: 220px;
```

Comment:
instead of using a margin-left which will perpetually put the error msgs 220px from the left, you could look into adding "col-sm-offset-_x_", (where x=3 since using bootstrap and the labels are col-sm-3 ) to the _obs-err-msg_  divs so that they stay in line with the input fields regardless of screen size

Reply:
Done. Had to add a padding though